src/dbnode/storage/index/aggregate_results.go

```diff
@@ -110,6 +190,14 @@ func (r *aggregatedResults) AddFields(batch []AggregateResultsEntry) (int, int)
 	r.Lock()
 	defer r.Unlock()
 
+	// NB: init total count with batch length, since each aggregated entry
+	// will have one field.
+	totalCount := len(batch)
+	for idx := 0; idx < len(batch); idx++ {
+		totalCount += len(batch[idx].Terms)
```

Comment:
how is total different than total terms?

Reply:
Removed this